CONTRIBUTING.md

```diff
@@ -81,3 +81,27 @@ SLACK_VERIFICATION_TOKEN=""
 ### Functional Tests
 
 1. Create a Slack team and add SlashDeploy to it using the button on `https://xxx.ngrok.io`
+
+### Postgres in Docker
```

Comment:
Can you take this out of this PR for now? A better way to do this would be to provide a `docker-compose.yml`.